.github/actions/setup/action.yml

```diff
@@ -0,0 +1,12 @@
+name: setup
+description: Setup Java and Gradle
+runs:
+  using: composite
+  steps:
+    - name: Setup Java
+      uses: actions/setup-java@v4
+      with:
+        distribution: 'temurin'
+        java-version: 8
```

Comment:
I think the `version3` is ready to merge. If that is merged first, this may need to be updated to Java 11.

Reply:
Cool! I didn't notice `version3` out there. I'll add some TODO comments, so hopefully it will be easier to remember the fix when the merge hits.  see: [227bbe5](https://github.com/sonatype-nexus-community/scan-gradle-plugin/pull/169/commits/227bbe5d19591928fd0be63d3df6744472d5f1b5)